src/functionalTest/java/uk/gov/hmcts/reform/cwrdapi/CaseWorkerRefFunctionalTest.java

```diff
@@ -359,11 +375,11 @@ public void shouldUploadServiceRoleMappingXlsFileSuccessfully() throws IOExcepti
     @ToggleEnable(mapKey = CASEWORKER_FILE_UPLOAD, withFeature = true)
     @ExtendWith(FeatureToggleConditionExtension.class)
     @Order(3)
-    public void shouldUploadServiceRoleMappingAba1XlsFileSuccessfully() throws IOException {
+    void shouldUploadServiceRoleMappingAba1XlsFileSuccessfully() throws IOException {
         ExtractableResponse<Response> uploadCaseWorkerFileResponse =
                 uploadCaseWorkerFile("src/functionalTest/resources/ServiceRoleMapping_ABA1.xls",
                         200, IDAM_ROLE_MAPPINGS_SUCCESS, TYPE_XLS,
-                        ROLE_CWD_ADMIN);
+                        ROLE_CWD_ADMIN,Boolean.TRUE);
```

Comment:
you can set this to false as for service to idam mapping file there is no need to update version